src/metabase/driver/crate/query_processor.clj

```diff
@@ -0,0 +1,18 @@
+(ns metabase.driver.crate.query-processor
+  (:require [korma.core :as k]
+            [metabase.driver.generic-sql.query-processor :as qp])
+  (:import (metabase.query_processor.interface ComparisonFilter CompoundFilter)))
+
+
+(defn- rewrite-between
+  "Rewrite [:between <field> <min> <max>] -> [:and [:>= <field> <min>] [:<= <field> <max>]]"
+  [clause]
+  (CompoundFilter. :and [(ComparisonFilter. :>= (:field clause) (:min-val clause))
+                         (ComparisonFilter. :<= (:field clause) (:max-val clause))]))
```

Comment:
instead of using the constructor directly `CompoundFilter.` it's preferable to use the `map->CompoundFilter` or `strict-map->CompoundFilter` functions because they're more readable and future-proof